src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV6.sql

```diff
@@ -148,6 +148,14 @@ BEGIN
     AND     SopInstanceKey3 = ISNULL(@instanceKey, SopInstanceKey3)
     AND     PartitionKey = @partitionKey
     AND     ResourceType = @imageResourceType
+
+    INSERT INTO dbo.ChangeFeed
```

Comment:
Can we keep this at the end of the sproc where it was? I want to minimize the length of the time that this table is locked to prevent too much contention

Reply:
Perhaps we can save the deleted rows below instead